duckduckgo/iOS
# Feature/migration

Reviewer: Mia
Asana: https://app.asana.com/0/392891325557410/390989879439961
CC: Caine

**Description**:
Migrates the users saved searches and stories from the Search and Stories app in to bookmarks in the new app. 

**Steps to test this PR**:
1. Checkout the Search and Stories app and rename the bundle identifier to match the new app's bundle identifier (discard this change once testing is complete - we'll be updating the bundle identifier of the new app match the for release)
2. Launch the Search and Stories app from Xcode 
3. Favourite one or more stories and make a note of what they are
4. Favourite one or more searches and make a note of what they are
5. Launch the new app from Xcode
6. Finish the onboarding screens and check the favourited stories and searches are present in the bookmarks


###### Reviewer Checklist:


###### PR DRI Checklist:
